.github/trivy-triage.yaml

```diff
@@ -0,0 +1,16 @@
+name: trivy triage
```

Comment:
```suggestion
name: Auto-label Discussion
```

Reply:
the reason I called it triage and not something more specific is I imagined once this is implemented we might add more automation in addition to labeling issues. or, if we did add more automation to discussions/issues, it will likely be in this workflow. do you prefer that I change it now to something more descriptive? I don't mind

Reply:
How about this?

```suggestion
name: Triage Discussion
```